.github/ISSUE_TEMPLATE/bugs_features.md

```diff
@@ -0,0 +1,35 @@
+<!--
+Thanks for your contribution!
+Please fill out the necessary sections, and delete unused ones.
+-->
+
+## Type of Issue:
+- [ ] Bug report.
+- [ ] Feature request (change which adds functionality).
+- [ ] Code refactor.
+- [ ] Documentation change.
```

Comment:
I find using checkboxes here is a bit weird because it means Github will never mark the issue as having 100% of the tasks completed since we have to choose only one of the four (as opposed to the Checklist)
This comment is probably also valid for PRs

Reply:
That's true. I find I have to delete the other checkboxes because of that.I will change it.